x-pack/platform/test/fleet_api_integration/config.base.ts

```diff
@@ -109,6 +109,9 @@ export default async function ({ readConfigFile, log }: FtrConfigProviderContext
             appenders: ['default'],
           },
         ])}`,
+        `--xpack.task_manager.unsafe.exclude_task_types=${JSON.stringify([
```

Comment:
That task is not used in our e2e test and could cause race condition with the added test